devops/actions/run-tests/benchmark/action.yml

```diff
@@ -140,6 +140,13 @@ runs:
     shell: bash
     run: |
       git clone -b unify-ci https://github.com/intel/llvm-ci-perf-results
+  # Linux tools installed during docker creation may not match the self-hosted
+  # kernel version, so we need to install the correct version here.
+  - name: Install perf in version matching the host kernel
+    shell: bash
+    run: |
+      sudo apt-get update
+      sudo apt-get install -y linux-tools-$(uname -r)
```

Comment:
This will surely work well, but there already is an installation script that installs `linux-tools-generic`: https://github.com/intel/llvm/blob/sycl/devops/scripts/install_build_tools.sh#L29. Perhaps it would be a better place for handling this missing package? If one were to use the nightly image for local tests, no additional installation would be required.